extensions/sql-migration/src/api/azureApi.ts

```diff
@@ -0,0 +1,12 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+export interface Subscription {
+
+}
+
+export async function getAllSubscriptions(): Promise<Subscription[]> {
```

Comment:
What's this for?

Reply:
tfw that file should not have made its way in. I'll remove it.